components/dashboard/src/projects/ProjectSettings.tsx

```diff
@@ -132,98 +142,114 @@ export default function ProjectSettingsView() {
                     Update Name
                 </Button>
             </form>
-            <Heading2 className="mt-12">Prebuilds</Heading2>
-            <Subheading>Choose the workspace machine type for your prebuilds.</Subheading>
-            <div className="max-w-md">
-                <SelectWorkspaceClassComponent
-                    selectedWorkspaceClass={project.settings?.workspaceClasses?.prebuild}
-                    onSelectionChange={setWorkspaceClassForPrebuild}
+            <div>
+                <Heading2 className="mt-12">Prebuilds</Heading2>
+                <CheckboxInputField
+                    label="Enable prebuilds"
+                    hint="You need to have admin permissions on the repository in order to install a webhook for prebuilds."
```

Comment:
**suggestion**: Left out the most important part, copy! 🙈 Minor rephrase to a) abstract permissions (admin, maintainer, etc) (see https://github.com/gitpod-io/gitpod/issues/16852#issuecomment-1706451086), b) mention both create+manage as you'll need permissions to be able to remove that webhook on removal, and c) make it shorter and more concise.

```suggestion
                    hint="Requires permissions to create and manage repository webhooks."
```